### ISSUE TITLE
Preserve orientation in EXIF data

### ISSUE THREAD

**Issue by frankus** [lgbtq-technology/lgbtq-helper-app#37]
Photos in portrait uploaded from iOS show up sideways. 

Photos taken on iOS store the data in the native orientation of the sensor and add an EXIF tag to reflect how the final image should be rotated for display.

**Comment by aredridel** [comment]
Yup. There's also some metadata bugs with Slack, so metadata may need to be stripped after rotation.

**Comment by aredridel** [comment]
https://justmarkup.com/articles/2019-10-21-image-orientation/
